tpl/tplimpl/embedded/templates/opengraph.html

```diff
@@ -1,14 +1,14 @@
 <meta property="og:url" content="{{ .Permalink }}">
 
-{{- with or site.Title site.Params.title | plainify }}
+{{- with or site.Title site.Params.title | .RenderString | plainify | htmlUnescape }}
```

Comment:
I suggest we focus on fixing #12418 and not add other not obviously related.

I assume the HTML gets escaped the second time in

```html
content="{{ . }}"
```

Where is the first escape?

Also, a test for #12418 would help.